### PR TITLE
Feature/expdat 025 General Refactorings 

### LINKED ISSUE #48: EXPDAT-025 General Refactorings
# user story

**As a** developer

**I want** to work on clean, uniform code

**so that** I can read it, maintain it and extend it better

# Acceptance criteria
- [ ] runs local without errors
- [ ] commit message contains issue abbreviations
- [ ] actually improves readability, maintainability, and extensibility

### PR DESCRIPTION
**Pull request by florianmarcher**
reformatted ugly code, is now beautiful :D #48

### CODE REVIEW

File: app/src/main/java/at/sw21_tug/team_25/expirydates/data/ExpItemDao.kt [hunk 1]
```diff
@@ -26,6 +26,6 @@ interface ExpItemDao {
     @Query("DELETE FROM items WHERE id = :itemId")
     suspend fun deleteItemById(itemId: Int)
 
-    @Update()
+    @Update
     suspend fun updateItem(item: ExpItem)
 }
```
Comment (steschi) [inline app/src/main/java/at/sw21_tug/team_25/expirydates/data/ExpItemDao.kt]: missing newline at EOF

File: app/src/main/java/at/sw21_tug/team_25/expirydates/ui/detailview/ui/DetailView.kt [hunk 2]
```diff
@@ -149,18 +158,26 @@ class DetailView(private val view: View) : DatePickerDialog.OnDateSetListener {
...
+
     override fun onDateSet(view: DatePicker?, year: Int, month: Int, dayOfMonth: Int) {
 
         val formatter = DateTimeFormatter.ofPattern("dd-MM-yyyy")
-        this.view.findViewById<TextView>(R.id.exp_date).text = LocalDate.of(year, month + 1, dayOfMonth).format(formatter)
+        this.view.findViewById<TextView>(R.id.exp_date).text =
+            LocalDate.of(year, month + 1, dayOfMonth).format(formatter)
     }
 
 }
```
Comment (steschi) [inline app/src/main/java/at/sw21_tug/team_25/expirydates/ui/detailview/ui/DetailView.kt]: missing newline at EOF

File: app/src/main/java/at/sw21_tug/team_25/expirydates/ui/list/ListViewModel.kt [hunk 3]
```diff
@@ -11,5 +11,5 @@ class ListViewModel : ViewModel() {
         value = ""
     }
     val text: LiveData<String> = _text
-    var expItems: LiveData<List<ExpItem>>?=null
+    var expItems: LiveData<List<ExpItem>>? = null
 }
```
Comment (steschi) [inline app/src/main/java/at/sw21_tug/team_25/expirydates/ui/list/ListViewModel.kt]: missing newline at EOF

File: app/src/main/java/at/sw21_tug/team_25/expirydates/utils/ReminderScheduler.kt [hunk 4]
```diff
@@ -29,36 +28,45 @@ object ReminderScheduler {
...
+            LocalDate.parse(dueDate, DateTimeFormatter.ISO_LOCAL_DATE).atStartOfDay()
         reminderDateTime = reminderDateTime.minusDays(1)
         reminderDateTime = reminderDateTime.withHour(9)
 
-        return reminderDateTime.toInstant(ZoneOffset.UTC).toEpochMilli() - currentDateTime.toInstant(ZoneOffset.UTC).toEpochMilli()
+        return reminderDateTime.toInstant(ZoneOffset.UTC)
+            .toEpochMilli() - currentDateTime.toInstant(ZoneOffset.UTC).toEpochMilli()
     }
 
 }
```
Comment (steschi) [inline app/src/main/java/at/sw21_tug/team_25/expirydates/utils/ReminderScheduler.kt]: missing newline at EOF

File: app/src/main/res/layout/fragment_detail_view.xml [hunk 5]
```diff
@@ -1,104 +1,98 @@
...
+            android:id="@+id/closePopUp"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="start"
+            android:layout_marginHorizontal="15dp"
+            android:text="@string/close"
+            tools:ignore="ButtonStyle" />
+    </LinearLayout>
 
 </FrameLayout>
```
Comment (steschi) [inline app/src/main/res/layout/fragment_detail_view.xml]: missing newline at EOF

File: app/src/main/res/layout/fragment_home.xml [hunk 6]
```diff
@@ -10,6 +9,5 @@
         android:id="@+id/map"
         android:name="com.google.android.gms.maps.SupportMapFragment"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        />
+        android:layout_height="match_parent" />
 </androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (steschi) [inline app/src/main/res/layout/fragment_home.xml]: missing newline at EOF

File: app/src/main/res/layout/fragment_list.xml [hunk 7]
```diff
@@ -6,14 +6,14 @@
...
+        android:id="@+id/items_rv"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
 
 </androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (steschi) [inline app/src/main/res/layout/fragment_list.xml]: missing newline at EOF

File: app/src/main/res/layout/item_list.xml [hunk 8]
```diff
@@ -1,14 +1,14 @@
...
     <TextView
         android:id="@+id/item_tv"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:textSize="20sp"
-        android:padding="20dp"/>
+        android:padding="20dp"
+        android:textSize="20sp" />
 
 </LinearLayout>
```
Comment (steschi) [inline app/src/main/res/layout/item_list.xml]: missing newline at EOF

File: app/src/main/res/menu/bottom_nav_menu.xml [hunk 9]
```diff
@@ -14,6 +14,6 @@
     <item
         android:id="@+id/navigation_list"
         android:icon="@drawable/ic_format_list_bulleted_24dp"
-        android:title="@string/title_list"/>
+        android:title="@string/title_list" />
 
 </menu>
```
Comment (steschi) [inline app/src/main/res/menu/bottom_nav_menu.xml]: missing newline at EOF

File: app/src/main/res/menu/language_choice_menu.xml [hunk 10]
```diff
@@ -1,9 +1,11 @@
...
-    <item android:id="@+id/language_en"
+    <item
+        android:id="@+id/language_en"
         android:title="EN" />
-    <item android:id="@+id/language_ru"
+    <item
+        android:id="@+id/language_ru"
         android:title="RU" />
 
 </menu>
```
Comment (steschi) [inline app/src/main/res/menu/language_choice_menu.xml]: missing newline at EOF

File: app/src/main/res/mipmap-anydpi-v26/expicon.xml [hunk 11]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
-    <background android:drawable="@drawable/expicon_background"/>
-    <foreground android:drawable="@mipmap/expicon_foreground"/>
+    <background android:drawable="@drawable/expicon_background" />
+    <foreground android:drawable="@mipmap/expicon_foreground" />
 </adaptive-icon>
```
Comment (steschi) [inline app/src/main/res/mipmap-anydpi-v26/expicon.xml]: missing newline at EOF

File: app/src/main/res/mipmap-anydpi-v26/expicon_round.xml [hunk 12]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
-    <background android:drawable="@drawable/expicon_background"/>
-    <foreground android:drawable="@mipmap/expicon_foreground"/>
+    <background android:drawable="@drawable/expicon_background" />
+    <foreground android:drawable="@mipmap/expicon_foreground" />
 </adaptive-icon>
```
Comment (steschi) [inline app/src/main/res/mipmap-anydpi-v26/expicon_round.xml]: missing newline at EOF

File: app/src/main/res/values-ru/dimens.xml [hunk 13]
```diff
@@ -1,5 +1,3 @@
 <resources>
     <!-- Default screen margins, per the Android Design guidelines. -->
-    <dimen name="activity_horizontal_margin">16dp</dimen>
-    <dimen name="activity_vertical_margin">16dp</dimen>
 </resources>
```
Comment (steschi) [inline app/src/main/res/values-ru/dimens.xml]: missing newline at EOF

File: app/src/main/res/values/dimens.xml [hunk 14]
```diff
@@ -1,5 +1,3 @@
 <resources>
     <!-- Default screen margins, per the Android Design guidelines. -->
-    <dimen name="activity_horizontal_margin">16dp</dimen>
-    <dimen name="activity_vertical_margin">16dp</dimen>
 </resources>
```
Comment (steschi) [inline app/src/main/res/values/dimens.xml]: missing newline at EOF

**Review by bengaal — APPROVED** [review]

**Review by steschi — APPROVED** [review]
tests pass, app compiles and works, very good code styling! 👍

**Pull request merged**